.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Apex Replay Debugger",
+            "type": "apex-replay",
+            "request": "launch",
+            "logFile": "${command:AskForLogFileName}",
+            "stopOnEntry": true,
+            "trace": true
+        },
+        {
+            "type": "node",
+            "name": "Debug Jest Tests",
+            "request": "launch",
+            "args": ["--runInBand", "--config=jest.config.js", "--silent"],
```

Comment:
`jest.config.js` is the default for Jest and automatically picked up.

Reply:
I personally don't like including the `silent` flag here. I know it significantly reduces noise from logs emitted by base components the user is unlikely to have any control over, but it also hides any `console.log` statements users may add themselves to help debug.

it's also inconsistent with how we run tests via our npm script in package.json. Any specific reason we chose to use silent here only?

Reply:
Main reason is the noise from the base components, as that makes the output nearly unreadable on the console.

As we're saying (via the eslint rules) that people shouldn't use console.log I think it's not a big issue to not show it. And rather point people how to properly debug, or?

Reply:
Could the advice for users be to use `console.warn` messages? Those come through.

Reply:
I'm less concerned with how users can get around this and more worried that things won't behave as expected for users and it won't be obvious why. Especially since these Jest options are different from what's in package.json.

We should either run with the `--silent` flag everywhere and document it saying we run with that flag to mask unnecessary warnings and as a side-effect your `console.log`s will also be masked, or run without the flag everywhere and let users hide the logs themselves if they want.

Should we add a section in the README about testing?

Reply:
Agreed. Let's remove `silent`. At some point the base components will get rid of their error messages... And I think Diego & Co. are looking into relaxing a bunch of the core engine warnings anyway.